tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TO_TAG="$1"
+
+FROM_TAG=$(grep "docker.io/rook/ceph" deploy/examples/images.txt | awk -F : '{ print $2 }')
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SED="${scriptdir}/../../build/sed-in-place"
```

Comment:
This raises a good point. `SED` is defined with an absolute path, which is good form. I can foresee someone running this script from the `tests/scripts` dir and then it not working correctly. (aside: @travisn should this reside next to `sed-in-place` in the build dir instead? IMO yes.)

Given that, I would suggest a different strategy that will ensure the whole script works, not just the sed portion.

```suggestion
scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
rook_root="${scriptdir}/../..}"
cd "${rook_root}" # allow using relative paths in script

SED="build/sed-in-place"
```